### DISCUSSION TITLE
Show-stopping error after one successful query

### DISCUSSION THREAD

**Discussion by thebundok** [imartinez/privateGPT#449] [Q&A] [answered]
Installation was successful, no errors. I injested one 378 page PDF, again no error. I made a single query, and while it took a while to respond it did so fairly accurately, and provided sources. 

I then asked a follow-on question and was hit with a wall of text ending with "ValueError: Requested tokens exceed context window of 1000". It also immediately terminated the session. I restarted the session, but each new query now results in the same error and it will no longer answer questions. Even closing the command prompt window and starting from scratch no longer provides answers to queries.

Searching around, it seems to have something to do with llama_tokenize, but I haven't found any solutions relevant to this particular issue. Nor do I understand the wall of text enough to be able to discern a possible solution myself. Whether resetting the available tokens or increasing the allowed amount. You can see the wall of text here:

[Tokens Error.txt](https://github.com/imartinez/privateGPT/files/11553401/Tokens.Error.txt)

I'm on an Intel Xeon E5-1650 v4 @ 3.60GHz and 64GB Ram. I'm using the Vicuna 13B parameters model.

**Comment by johnbrisbin** [comment, marked as answer]
With some models it is possible to increase the 
MODEL_N_CTX=1000
in the .env file to a larger number. Give 2000 or some other number a try, quit privateGPT.py and go again.

**Reply by thebundok** [reply]
Thanks. This seems to have solved it for the time being. I've changed the MODEL_N_CTX=2000 and so far the error has gone away. Thank you for the quick, and helpful, response. :)
